spec/forms/idv/finance_form_spec.rb

```diff
@@ -5,21 +5,21 @@
 
   describe 'presence validations' do
     it 'is invalid when required attributes are not present' do
-      valid_params = { finance_type: :ccn, finance_account: '12345678' }
+      valid_params = { finance_type: :ccn, ccn: '12345678' }
```

Comment:
It appears that this spec file does not have any specs for non-`ccn` finance types. Should we add some?